components/core/CarouselSidebarSlate.js

```diff
@@ -337,8 +366,14 @@ export default class CarouselSidebarSlate extends React.Component {
               placeholder="Author"
               onChange={this._handleChange}
               id={`sidebar-label-author`}
-              style={{ ...STYLES_INPUT, marginBottom: 24 }}
+              style={{ ...STYLES_INPUT, marginBottom: 12 }}
             />
+            {this.state.unsavedChanges == false && (
+              <div css={STYLES_AUTOSAVE}>
+                <SVG.Check height="14px" style={{ marginRight: 4 }} />
+                autosaved
```

Comment:
Lets improve the english here and add a timestamp in the next pass.

Reply:
sure I can work on improving this